sdk/storage/storage-blob/test/pageblobclient.spec.ts

```diff
@@ -146,10 +146,10 @@ describe("PageBlobClient", () => {
     await pageBlobClient.clearPages(512, 512);
 
     const rangesDiff = await pageBlobClient.getPageRangesDiff(0, 1024, snapshotResult.snapshot!);
-    assert.equal(rangesDiff.pageRange![0].start, 0);
-    assert.equal(rangesDiff.pageRange![0].end, 511);
-    assert.equal(rangesDiff.clearRange![0].start, 512);
-    assert.equal(rangesDiff.clearRange![0].end, 1023);
+    assert.equal(rangesDiff.pageRange![0].offset, 0);
+    assert.equal(rangesDiff.pageRange![0].offset + (rangesDiff.pageRange![0].count || 0), 511);
```

Comment:
nit: I know it's from original test but we could also verify the `.count` directly.